Tasks/AzureRmWebAppDeploymentV4/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 4,
         "Minor": 3,
-        "Patch": 5
+        "Patch": 8
```

Comment:
Again is this the patch version we want?

Reply:
we can skip this.

Reply:
thanks, updated.